test-workloads/src/bin/tellus.rs

```diff
@@ -138,6 +140,102 @@ fn exercise_pmu_functionality() {
     .expect_err("Successfully configured FW counter");
 }
 
+#[cfg(target_feature = "v")]
+fn store_into_vectors() {
+    let vec_len: u64 = 8;
+    let vtype: u64 = 0xda;
+    let enable: u64 = 0x200;
+
+    println!("Writing tellus vector registers");
+    unsafe {
+        // safe because we are only setting the vector csr's
+        asm!(
+        "csrrs zero, sstatus, {enable}",
+
+        "vsetvl x0, {vec_len}, {vtype}",
+        vec_len = in(reg) vec_len,
+        vtype = in(reg) vtype,
+        enable = in(reg) enable,
+        options(nostack),
+        )
+    }
+
+    const REG_WIDTH_IN_U64S: usize = 4;
+
+    let mut inbuf = [0_u64; (32 * REG_WIDTH_IN_U64S)];
+    for i in 0..inbuf.len() {
+        inbuf[i] = 0xDEADBEEFCAFEBABE;
+    }
+
+    let bufp1 = inbuf.as_ptr();
+    let bufp2: *const u64;
+    let bufp3: *const u64;
+    let bufp4: *const u64;
+    unsafe {
+        // safe because we don't go past the length of inbuf
+        bufp2 = bufp1.add(8 * REG_WIDTH_IN_U64S);
+        bufp3 = bufp1.add(16 * REG_WIDTH_IN_U64S);
+        bufp4 = bufp1.add(24 * REG_WIDTH_IN_U64S);
+    }
+    unsafe {
+        // safe because the assembly reads into the vector register file
+        asm!(
+        "vl8r.v  v0, ({bufp1})",
+        "vl8r.v  v8, ({bufp2})",
+        "vl8r.v  v16, ({bufp3})",
+        "vl8r.v  v24, ({bufp4})",
+        bufp1 = in(reg) bufp1,
+        bufp2 = in(reg) bufp2,
+        bufp3 = in(reg) bufp3,
+        bufp4 = in(reg) bufp4,
+        options(nostack));
```

Comment:
Looks like `cargo fmt` isn't complaining, but let's be consistent about how the `asm!` blocks are formatted. Something like:

```
asm!(
    "vl84.v v0, ({bufp1})",
    ...
    options(nostack),
);
```

Reply:
will fix